metalnetlet/controllers/network_controller.go

```diff
@@ -144,6 +144,16 @@ func (r *NetworkReconciler) reconcile(ctx context.Context, log logr.Logger, netw
 			ID: vni,
 		},
 	}
+
```

Comment:
I think here we are missing the `Applied` state update of the `Network` peering status. `apinetlet` is patching the `Pending` state once peering refs are set. And once the `metalnetlet` is patching the corresponding `metalnet` `Network` resource it should patch the status to `Applied`.

Reply:
I think we need to wait for that to add `NetworkPeeringStatus` fields in `metalnet`